resources/.eslintrc-standards--annoy-mode.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "rules": {
+        "no-magic-numbers": ["error", {                          // disallow magic numbers
+            "ignore": [0, 1, -1],
```

Comment:
Those are the sorts of numbers I use and like...  But if we start adding those, where does this rule end in growth?